src/test/shamap/SHAMapSync_test.cpp

```diff
@@ -98,7 +98,7 @@ class SHAMapSync_test : public beast::unit_test::suite
         for (int i = 0; i < items; ++i)
         {
             source.addItem(
-                SHAMapNodeType::tnACCOUNT_STATE, std::move(*makeRandomAS()));
+                SHAMapNodeType::tnACCOUNT_STATE, std::move(makeRandomAS()));
```

Comment:
Remove `std::move` to allow the temporary `intrusive_ptr` to bind directly to the `addItem` argument.

Reply:
Resolved.